AppCenter/AppCenter/Internals/Ingestion/MSHttpIngestion.m

```diff
@@ -226,7 +226,7 @@ - (void)sendCallAsync:(MSIngestionCall *)call {
     NSURLSessionDataTask *task = [self.session
         dataTaskWithRequest:request
           completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
-            @synchronized(self) {
+            @synchronized(@"sendCallAsync") {
```

Comment:
Can you explain using NSString as a monitor? It doesn't look like correct code at all.

Reply:
I just tried，still analyzing the problem really，from the log point of view is the crash in this method.

Reply:
Maybe a thread problem.